main/src/cgeo/geocaching/connector/gc/GCMap.java

```diff
@@ -367,6 +358,56 @@ public String apply(final Bitmap bitmap, final String data) {
         return searchResult;
     }
 
+    /**
+     * Searches the view port on the live map for caches.
+     * The strategy dictates if only live map information is used or if an additional
+     * searchByCoordinates query is issued.
+     *
```

Comment:
I think the commented is C&P as "strategy" is no longer used in your code AFAICS.

Reply:
Good catch, I'm gonna do a bit more cleanup.